### PR TITLE
samples: nfc: Disable unused SPI instance

### PR DESCRIPTION
**Pull request by KAGA164**
Disable unused SPI instance in samples which use
ST25R3911B NFC shield to avoid interaction with
not used pins on nrf52840 pca10056 which has
enabled all SPI instance by default.

Signed-off-by: Kamil Gawor <Kamil.Gawor@nordicsemi.no>

### CODE REVIEW

File: samples/nfc/tag_reader/prj.conf [hunk 1]
```diff
@@ -9,6 +9,11 @@ CONFIG_SPI_0=y
 CONFIG_SPI=y
 CONFIG_SPI_0_NRF_SPI=y
 
+#Disable unused SPI instance
+CONFIG_SPI_1=n
```
Comment (kapi-no) [inline samples/nfc/tag_reader/prj.conf]: This instance is used on nRF5340. It seems that UART0 (used for logging) & SPI0 share HW resources and they cannot be used simultaneously on nRF5340.

Comment (KAGA164) [reply samples/nfc/tag_reader/prj.conf]: nRF5340 has a different dedicated `.conf` file. This changes does not have any impact on it

Comment (kapi-no) [reply samples/nfc/tag_reader/prj.conf]: yes, you are right

**Comment by carlescufi** [comment]
@KAGA164 please rebase

**Comment by carlescufi** [comment]
@KAGA164 CI is failing, can you check?

**Comment by kapi-no** [comment]
CI is green, this can be merged

**Pull request merged**